resources/winsetup/Yarn.wxs

```diff
@@ -36,6 +36,15 @@
 					Action="set"
 					System="yes"
 				/>
+				<Environment
+					Id="BinPath"
+					Name="PATH"
+					Value="[LocalAppDataFolder]Yarn\.bin"
+					Permanent="no"
+					Part="last"
+					Action="set"
+					System="yes"
```

Comment:
This is a per-user path, so other users in the same system don't have access to it.

This needs to be set up for each user that runs yarn. This will work on single-user systems, though.

An advantage of making it not System is that it will roam with the user profile. While the local appdata folder won't roam, local installs with yarn will go to the local appdata, so they'll show up in the path once you create the folder with yarn.